### PR TITLE
Added 11th Feb talk

### PR DESCRIPTION
**Pull request by asgrim**

### CODE REVIEW

File: data/meetups/2015-02-11.php [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+$eventbriteWidget .= '</div></div>';
+
+$meetup = new MeetupEntity();
+
+$abstract = <<<END
+A summary of successful testing for business platforms mostly built with PHP.  The return on investment for testing  Many of these are network services, and all require high levels of uptime.  The practices of TDD and BDD are discussed; and realistic scenarios walked through.  I will branch out into related areas such as CI depending on time used.  Material covered should be standard practice for professional developers.
+END;
+
+$meetup->setId(0)
+    ->setFromDate(new DateTime('2015-01-11 19:00'))
```
Comment (lboynton) [inline data/meetups/2015-02-11.php]: Still says January :stuck_out_tongue:

File: data/meetups/2015-02-11.php [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    ->setToDate(new DateTime('2015-01-11 23:00'))
+    ->setRegistrationUrl("http://www.eventbrite.co.uk/event/{$eid}")
+    ->setLocationUrl("https://www.google.co.uk/maps?q=Oasis+Venue,+Arundel+Street,+PO1+1NP&hl=en&ll=50.799642,-1.086724&spn=0.011772,0.031629&sll=50.799734,-1.086874&sspn=0.011772,0.031629&hq=Oasis+Venue,&hnear=Arundel+St,+PO1+1NP,+United+Kingdom&t=m&z=16")
+    ->setLocation('Oasis the Venue, Arundel Street, PO1 1NP')
+    ->setTalkingPoints(array(
+    	new TalkEntity('Owen Beresford', 'ChannelOwen', 'Test Strategies && Process', $abstract),
```
Comment (lboynton) [inline data/meetups/2015-02-11.php]: Double && here, should be &

Comment (asgrim) [inline data/meetups/2015-02-11.php]: Nope, that's the actual talk title

File: data/meetups/2015-02-11.php (hunk 1 above)
Comment (asgrim) [inline data/meetups/2015-02-11.php]: Will fix - thanks :)

File: data/meetups/2015-02-11.php (hunk 2 above)
Comment (lboynton) [inline data/meetups/2015-02-11.php]: Hmm ok, programming &&?

Comment (asgrim) [inline data/meetups/2015-02-11.php]: It should, however be `&amp;&amp;` :)

Comment (lboynton) [inline data/meetups/2015-02-11.php]: Touché ;)

**Pull request merged**